pkg/diagnoser/diagnoser.go

```diff
@@ -36,9 +39,13 @@ func (b *BaseDiagnoser) AddToDiagnoserFiles(file string) {
 
 // Export implements the interface method
 func (b *BaseDiagnoser) Export() error {
-	if b.exporter != nil {
-		return b.exporter.Export(b.diagnoserFiles)
+	var result error
+	for _, exporter := range b.exporters {
+		if exporter != nil {
```

Comment:
not sure this if is very usefull
also, I think we could use `errgroup.Group` here and run each exporter in a go func

Reply:
Yeah not sure it's useful either, just matching the existing shape for now. Can possibly remove this later or change it as you suggest